addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/handler/PicoKeypadHandler.java

```diff
@@ -0,0 +1,115 @@
+
+import org.eclipse.smarthome.core.thing.Thing;
+import org.openhab.binding.lutron.internal.KeypadComponent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Handler responsible for communicating with Lutron Pico keypads
+ *
+ * @author Bob Adair - Initial contribution
+ */
+public class PicoKeypadHandler extends BaseKeypadHandler {
+
+    private static enum Component implements KeypadComponent {
+        // Buttons for 2B, 2BRL, 3B, and 3BRL models
+        BUTTON1(2, "button1", "Button 1"),
+        BUTTON2(3, "button2", "Button 2"),
+        BUTTON3(4, "button3", "Button 3"),
+
+        RAISE(5, "buttonraise", "Raise Button"),
+        LOWER(6, "buttonlower", "Lower Button"),
+
+        // Buttons for PJ2-4B model
+        BUTTON1_4B(8, "button01", "Button 1"),
+        BUTTON2_4B(9, "button02", "Button 2"),
+        BUTTON3_4B(10, "button03", "Button 3"),
+        BUTTON4_4B(11, "button04", "Button 4");
+
+        private final int id;
+        private final String channel;
+        private final String description;
+
+        Component(int id, String channel, String description) {
+            this.id = id;
+            this.channel = channel;
+            this.description = description;
+        }
+
+        @Override
+        public int id() {
+            return id;
+        }
+
+        @Override
+        public String channel() {
+            return channel;
+        }
+
+        @Override
+        public String description() {
+            return description;
+        }
+    }
+
+    private final Logger logger = LoggerFactory.getLogger(PicoKeypadHandler.class);
+
+    public PicoKeypadHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    protected boolean isLed(int id) {
+        return false; // No LEDs on Picos
+    }
+
+    @Override
+    protected boolean isButton(int id) {
+        return (id >= 2 && id <= 11);
+    }
+
+    @Override
+    protected boolean isCCI(int id) {
+        return false;
+    }
+
+    @Override
+    protected void configureComponents(String model) {
+        String mod = model == null ? "null" : model;
+        logger.debug("Configuring components for keypad model {}", mod);
+
+        switch (mod) {
+            case "2B":
+                buttonList = Arrays.asList(Component.BUTTON1, Component.BUTTON3);
+                break;
+            case "2BRL":
+                buttonList = Arrays.asList(Component.BUTTON1, Component.BUTTON3, Component.RAISE, Component.LOWER);
+                break;
+            case "3B":
+                buttonList = Arrays.asList(Component.BUTTON1, Component.BUTTON2, Component.BUTTON3);
+                break;
+            case "4B":
+                buttonList = Arrays.asList(Component.BUTTON1_4B, Component.BUTTON2_4B, Component.BUTTON3_4B,
+                        Component.BUTTON4_4B);
+                break;
+            default:
+                logger.warn("No valid keypad model defined ({}). Assuming model 3BRL.", mod);
```

Comment:
If this is an optional configuration parameter it is by design so it should be logged on debug instead.

Reply:
I've updated the logic to be the same for all three configurable keypads.  Setting no model parameter defaults to "Generic".  Setting an incorrect model parameter value issues a warning and defaults to "Generic".

Reply:
That's a good improvement and I agree the warning can stay!